library/src/main/java/org/codice/security/sign/SystemCrypto.java

```diff
@@ -42,16 +42,20 @@
 
   private final String encryptionAlias;
 
-  public SystemCrypto() throws IOException {
+  public SystemCrypto(String hostname) throws IOException {
     this.passwordEncryption = null;
     // new EncryptionServiceImpl()
 
-    Properties sigProperties = createProperty("encryption.properties");
+    if (hostname == null) {
+      hostname = "samlhost";
```

Comment:
I will make this a constant somewhere. Something like `DEFAULT_HOSTNAME` or something.